operator/auth/spire/client.go

```diff
@@ -135,7 +135,7 @@ func (c *Client) onStart(_ hive.HookContext) error {
 				c.entry = entryv1.NewEntryClient(conn)
 				break
 			}
-			c.log.WithError(err).Errorf("Unable to connect to SPIRE server, attempt %d", attempts+1)
+			c.log.WithError(err).Warnf("Unable to connect to SPIRE server, attempt %d", attempts+1)
```

Comment:
While a warning is surely more appropriate than an error, reading the linked issue, it seems that we could have similar downsides with a warning too. Why not setting something like a reasonable threshold (either in terms of time or retries count) after which starts logging the warning?

Reply:
I have been thinking about this but finding a certain time to assume a bootstrap to be ready is hard to define as it depends on many factors. It being not connected is already surfaced in cilium status. 
Suggestions on this welcome, i like the idea but afraid to put a number on it

Reply:
Yep, I see your point and TBH I don't feel comfortable suggesting a number, either. A bit too verbose still sounds better than being too silent, so I'd say let's go with this and see if it'll be possible to define a reasonable threshold later, if needed. :+1: